src/flowgger/splitter/syslen_splitter.rs

```diff
@@ -11,28 +11,24 @@ impl<T: Read> Splitter<T> for SyslenSplitter {
     fn run(&self, buf_reader: BufReader<T>, tx: SyncSender<Vec<u8>>, decoder: Box<Decoder>, encoder: Box<Encoder>) {
         let mut buf_reader = buf_reader;
         loop {
-            if let Err(e) = read_msglen(&mut buf_reader) {
+            let size = match read_msglen(&mut buf_reader) {
+                Ok(size) => size,
+                Err(_) => {
+                    let _ = writeln!(stderr(), "Can't read message's length");
+                    return
+                },
+            };
+            println!("{}", size);
```

Comment:
Looks good! Can you just remove this `println!()`? Probably a leftover just used for debugging :)